projects/exui-common-lib/src/lib/components/write-address-inputs/write-address-inputs.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { WriteAddressInputsComponent } from './write-address-inputs.component';
+
+describe('WriteAddressFieldComponent', () => {
+
+  @Component({
+    selector: `write-address-inputs`,
+    template: `<xuilib-write-address-inputs [formGroup]="formGroup" [isInternational]="isInternational">
+    </xuilib-write-address-inputs>`
+  })
+  class WrapperComponent {
+    @ViewChild(WriteAddressInputsComponent, /* TODO: add static flag */ {})
+    public componentUnderTest: WriteAddressInputsComponent;
+    public formGroup = addressFormGroup();
+    public isInternational: boolean;
+  }
+  let wrapperComponent: any;
+  let fixture: ComponentFixture<WrapperComponent>;
+
+  function addressFormGroup() {
```

Comment:
@johnbenjamin-hmcts - could do with a return type but not a major thing, given it's only for tests